openparticle/src.py

```diff
@@ -1000,6 +1054,15 @@ def max_mode(self):
                     maximum_mode = mode
         return maximum_mode
 
+    @property
+    def modes(self):
+        modes_list = []
+
+        for term in self.op_dict:
+            modes_list.append([key[1] for key in term])
+
+        return modes_list
```

Comment:
Correct me if I'm wrong, but this looks to return a list of all modes for all terms in an operator instead of a set of the modes.

So the following operator:
`ParticleOperator("b0 b1 b2") + ParticleOperator("b0 b2")`
would have `.modes == [0, 1, 2, 0, 2]`.

Is correct and is that the intended behavior?